wgpu/src/backend.rs

```diff
@@ -64,6 +64,7 @@ impl Backend {
     pub fn draw<T: AsRef<str>>(
         &mut self,
         device: &wgpu::Device,
+        staging_belt: &mut wgpu::util::StagingBelt,
```

Comment:
reiterating what I asked in another comment - any strong reason to use this versus `write_buffer`/`write_texture`?